cfgov/apache/conf.d/redirects.conf

```diff
@@ -13,6 +13,7 @@ RedirectMatch permanent ^/policy-compliance/amicus/(.*) /compliance/amicus/$1
 RedirectMatch permanent ^/policy-compliance/enforcement/(.*)  /enforcement/$1
 RedirectMatch permanent ^/policy-compliance/rulemaking/(.*)  /rules-policy/$1
 RedirectMatch permanent ^/policy-compliance/guidance/(.*)  /compliance/$1
+RedirectMatch permanent ^/owning-a-home/process/(.*)  /owning-a-home/$1
```

Comment:
I don't have my local Apache setup so can't test this right away, but will this intercept https://www.consumerfinance.gov/owning-a-home/process/? We have an existing Wagtail redirect that points that to the first step in the process (https://www.consumerfinance.gov/owning-a-home/prepare/), so this would change that.

<img width="1163" alt="image" src="https://user-images.githubusercontent.com/654645/179037088-26f4b0e9-69b4-4058-9a12-f17b53e1dd4c.png">

Additionally we have many other Wagtail redirects that point to /owning-a-home/process/something; should we update all of those as well?

Reply:
I have never understood that specific /prepare redirect and would prefer this override it (and remove the Wagtail one) for the sake of consistency.

Very on board with removing those other redirects!